graph/impl/KokkosGraph_Distance2MIS_impl.hpp

```diff
@@ -853,7 +853,7 @@ struct D2_MIS_FixedPriority {
       // Finally, flip the worklists
       std::swap(worklist1, worklist2);
       workRemain = newWorkRemain;
-      numIter++;
+      // numIter++; // AquiEEP
```

Comment:
Same as above?

Reply:
I am hopeful that a merge with develop will take out the variable 'numIter' automatically. I will keep this feedback open for now, until I merge with develop.

Reply:
Luc helped me with the merge with develop. Everything went smoothly, and it seems no one got this error with "numIter" before, so I am removing it, per Luc's suggestion.